extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -377,45 +396,57 @@ export class SqlDatabaseTree {
 	}
 
 	private createDescription(view: azdata.ModelView): azdata.FlexContainer {
-		const descriptionTitle = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+		const descriptionTitle = view.modelBuilder.text().withProps({
 			value: 'Description',
 			CSSStyles: {
 				'font-size': '14px',
-				'margin-block-start': '0px',
-				'margin-block-end': '0px'
+				'width': '200px',
+				'margin': '10px 35px 0px 0px'
 			}
 		}).component();
-		this._descriptionText = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+		this._descriptionText = view.modelBuilder.text().withProps({
 			value: 'It is a job step that runs a PowerShell scripts.',
 			CSSStyles: {
-				'font-size': '12px'
+				'font-size': '12px',
+				'width': '200px',
+				'margin': '3px 35px 0px 0px'
 			}
 		}).component();
 
-		const recommendationTitle = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({
+		const recommendationTitle = view.modelBuilder.text().withProps({
 			value: 'Recommendation',
```

Comment:
It would be fantastic to spend 15min and get these all localized (in a separate PR, I'm just going to keep bringing it up until it's fixed 😛 )